### DISCUSSION TITLE
Variables colors

### DISCUSSION THREAD

**Discussion by CA10C3SAR** [zero-plusplus/vscode-autohotkey-debug#98] [Q&A] [answered]
Is possible to change the variables colors in the vscode?
I mean the colors of `if,return,gui,guicontrol` etc

**Comment by zero-plusplus** [comment, marked as answer]
Note: I use machine translation.

This extension does not support syntax highlighting.

So you will need to install another extension to solve your issue.
I guess you are using [slevesque.vscode-autohotkey](https://marketplace.visualstudio.com/items?itemName=slevesque.vscode-autohotkey) since you are not getting variable coloring.
Uninstalling it first and installing [cweijan.vscode-autohotkey-plus](https://marketplace.visualstudio.com/items?itemName=cweijan.vscode-autohotkey-plus) instead should solve your issue.
